docs/usage.rst

```diff
@@ -257,6 +257,14 @@ each scenario instead of flushing the entire database. Tests run much quicker,
 however HTTP server is not started and therefore web browser automation is
 not available.
 
+``--runner``
+*******************
+
+Allows to override the test runner class that Behave uses by default. e.g.,
```

Comment:
The "e.g.," is lost at the end of this line here. Why not remove it here and use it properly in the next sentence?